UnitySetup/UnitySetup.psm1

```diff
@@ -1397,27 +1399,48 @@ function Get-UnityProjectInstance {
         [string] $BasePath = $PWD,
 
         [parameter(Mandatory = $false)]
-        [switch] $Recurse
+        [switch] $Recurse,
+
+        [parameter(Mandatory = $false, ParameterSetName = "RecurseUp")]
```

Comment:
We could include this behavior without restricting the others. Is there a strong reason to use the parameter set name?